test/Helper/Navigation/BreadcrumbsTest.php

```diff
@@ -233,17 +233,14 @@ public function testRenderingPartialBySpecifyingAnArrayAsPartial()
         $this->assertEquals($expected, $this->_helper->render());
     }
 
-    public function testRenderingPartialShouldFailOnInvalidPartialArray()
+    public function testRenderingPartialShouldFailOnInvalidPartialArray(): void
     {
         $this->_helper->setPartial(['bc.phtml']);
-
-        try {
-            $this->_helper->render();
-            $this->fail(
-                '$partial was invalid, but no Laminas\View\Exception\ExceptionInterface was thrown'
-            );
-        } catch (ExceptionInterface $e) {
-        }
+        $this->expectException(InvalidArgumentException::class);
+        $this->_helper->render();
+        $this->fail(
+            '$partial was invalid, but no Laminas\View\Exception\ExceptionInterface was thrown'
+        );
```

Comment:
I think calling the fail is not needed anymore.

Reply:
No… it's not needed but I thought leaving it would aid docs ?

Reply:
The call of `expectException` is enough already.